### DISCUSSION TITLE
GD32F405RGT6

### DISCUSSION THREAD

**Discussion by Mixa778** [micropython/micropython#11115] [STM32 / Pyboard]
Hello. Can you please tell me is it possible to use the GD32F405RGT6 microcontroller instead of the STM32F405RGT6 microcontroller?

**Comment by rkompass** [comment]
Googling revealed a YouTube video and reports that GD32F401 iirc. behaved as expected (like STM) when connected to a Segger or the like (hardware debugging probe) but did not work with standard bootloader.
I suppose it will be hard to find support here for these chips. That's due to the fact that the GD32F405RGT6 is an _unofficial_  clone of the STM32F405RGT6 (please correct me if I'm wrong). So it's stolen intellectual property. Why bother with the quality problems of such hardware?
It comes to mind also that Chinese Pyboards sold on AliExpress, for example, have a certain negative impact on the development of micropython in that they represent missing revenues that imply less paid developers... . Of course clones also contribute to the fame of a good product.

**Reply by bidrohini** [reply]
I came across some similar discussions in other forums too. Here is a video you can look at: https://www.youtube.com/watch?v=xej19edPU9M

I am agreed with rkompass regarding the quality problem that may arise.

**Comment by Mixa778** [comment]
Good afternoon. As far as I know the company bought a license from STM. Due to the crisis in the microelectronics market, the price tag for microcontrollers is very high, so it is necessary to look for alternatives for the development of Pyboards....

**Reply by rkompass** [reply]
Thanks @Mixa778 for the correction. So no stolen intellectual property. In this case it seems someone has to tryout whether MP runs.
